docs/Tutorial.md

```diff
@@ -0,0 +1,538 @@
+A strategy that works well in practice is to iterate the implementation so that it satisfies the simplest tests first, and then the more complex ones. That way we keep the room for introducing bugs tight, and debugging becomes easier.
+To define what is rendered, we use [JSX](https://reactjs.org/docs/introducing-jsx.html), an extension to JavaScript which allows us to define the structure of the document in a declarative way, and then Babel compiles it to DOM imperative calls of type document.createElement().
+Let's see how our component that passes the tests is implemented.
+
+```javascript
+// File Path: lime-plugin-remotesupport/src/remoteSupportPage.js
+import { h } from 'preact';
+import { useSession, useOpenSession, useCloseSession } from './remoteSupportQueries';
+import Loading from 'components/loading';
+// We use i18n-js for internazionalization
+import I18n from 'i18n-js';
+
+const RemoteSupportPage = () => {
+	// React Query make it easy to see the status of a query (i.e, isLoading).
+	const {data: session, isLoading: loadingSession} = useSession();
+	const [openSession, openStatus] = useOpenSession();
+	const [closeSession, closeStatus] = useCloseSession();
+
+	if (loadingSession) {
+		return <div class="container container-center"><Loading /></div>
+	}
+
+	return(
+		<div>
+			<h4>{I18n.t("Ask for remote support")}</h4>
+			{!session &&
+				<div>
+					<p>{I18n.t("There's no open session for remote support. Click at Create Session to begin one")}</p>
+					<button onClick={openSession}>{I18n.t("Create Session")}</button>
+				</div>
+			}
+			{openStatus.isError &&
+				<div>
+					<b>{I18n.t("Cannot connect to the remote support server")}</b><br />
+					{I18n.t("Please verify your internet connection")}
+				</div>
+			}
+			{session &&
+				<div>
+					<p>{I18n.t("There's an active remote support session")}</p>
+					<p>{I18n.t("Share the following command with whoever you want to give them access to your node")}</p>
+					<div><pre>{session.rw_ssh}</pre></div>
+					<div>
+						<h5>{I18n.t("Close Session")}</h5>
+						<p>{I18n.t("Click at Close Session to end the remote support session. No one will be able to access your node with this token again")}</p>
+						<button onClick={closeSession}>{I18n.t("Close Session")}</button>
+					</div>
+				</div>
+			}
+			{openStatus.isLoading || closeStatus.isLoading &&
+				<Loading />
+			}
+		</div>
+	);
+
+	export default RemoteSupportPage;
+```
+
+#### 4.3.1 Style and Visualization
+
+##### Storybook
+
+Once our component passes the tests, we can add styling with css classes to achieve the UI we are looking for.
+
+In order to visualize our component while we add style to it we use [StoryBook](https://storybook.js.org/). Storybook allows you to define the different states of your screen as different user `Stories` and render your component isolated in that context. Once the stories are written, it is easy to go back to them to test changes in the UI.
```

Comment:
maybe you can add some gif here showing the storybook feature... how it looks and why it is useful

Reply:
Nice, done!